apps/decoder_json/lib/decoder/json_lines.ex

```diff
@@ -10,13 +10,9 @@ defmodule Decoder.JsonLines do
   defimpl Decoder do
     def lines_or_bytes(_t), do: :line
 
-    def decode(_t, stream) do
-      stream
-      |> Stream.map(&decode_chunk/1)
-    end
-
-    defp decode_chunk(chunk) do
-      Enum.map(chunk, &Jason.decode!/1)
+    def decode(_t, messages) do
+      messages
+      |> Enum.map(&Jason.decode!/1)
```

Comment:
for a single function call, i wouldn't bother with the pipe here (see above in json_arrays)